layouts/partials/compliance/insights.html

```diff
@@ -0,0 +1,39 @@
+<section class="mt-8 px-4 md:px-0">
+    <div class="container mx-auto max-w-4xl">
+        <h3>Pulumi Insights</h3>
+        <p class="py-4">
+            Use <a href="/product/pulumi-insights/">Pulumi Insights</a> to gain visibility into your cloud infrastructure's
+            configuration to assess {{ .Params.framework }} compliance. Pulumi Insights is Intelligence for Infrastructure as Code.
+            It adds advanced search, analytics, and AI to infrastructure as code.
+        </p>
+        <img src="https://www.pulumi.com/uploads/pulumi-insights-search.gif" />
+        <p class="py-4">
+            Use Copilot to assist configuring your infrastructure to make it compliance ready. You can tap into the Pulumi Copilot's
+            deep understanding of your organization's context to gain visibility into the configuration of resources and assess their
+            compliance.
+        </p>
+        <img src="/images/compliance/co-pilot-1.png" />
+    </div>
+</section>
+
+
+<section class="mt-8 py-4 px-4 md:px-0">
+    <div class="container mx-auto max-w-4xl">
+        <h3>Compliance Ready Policies</h3>
+        <p class="py-4">
+            With comprehensive coverage of {{ .Params.cloud }},
+            <a href="/docs/using-pulumi/crossguard/compliance-ready-policies/">Pulumi Compliance Ready Policies</a>
+            provide an enhanced level of control and governance over your cloud resources. Pulumi Compliance Ready
+            Policies empower you to enforce best practices, security standards, cost control and compliance requirements
+            seamlessly within your infrastructure-as-code workflows.
+        </p>
+    </div>
+</section>
+
+<section id="sa-form" class="container mx-auto justify-center text-center m-4">
+    <div class="w-full flex flex-col items-center lg:w-1/2 card bg-white mx-auto p-8">
+        <h3>Talk to a Solutions Architect</h3>
```

Comment:
I would like to see these form be a lot wider and far less tall. Like half the height

Reply:
I think we might not be able to do this here. Hubspot will not allow me to make it any wider since it won't allow me to make the form more than 3 columns wide. It was however rendering taller on the pci pages, and I have addressed that, so check now and let me know what you think.

Reply:
Ok maybe the solution is put these things in 2 columns. Compliance for AWS services on the left and Talk to a Solutions Architect on the right. That way there isn't as much wasted whitespace